pkg/cookies/csrf_test.go

```diff
@@ -158,10 +158,9 @@ var _ = Describe("CSRF Cookie Tests", func() {
 				))
 				Expect(rw.Header().Get("Set-Cookie")).To(ContainSubstring(
 					fmt.Sprintf(
-						"; Path=%s; Domain=%s; Expires=%s; HttpOnly; Secure",
+						"; Path=%s; Domain=%s; HttpOnly; Secure",
```

Comment:
Why does the CSRF now not have an expiration?

Reply:
That is because `CSRFExpire` in options.Cookie  is not set here.
I will revoke this change and add CSRFExpire instead.

https://github.com/oauth2-proxy/oauth2-proxy/blob/f753ec1ca5c42cd4c4e400afa1f76817638f3010/pkg/cookies/csrf_test.go#L25-L34